frontstage/templates/surveys/surveys-todo.html

```diff
@@ -16,6 +16,15 @@
 </div>
 {% endif %}
 
+
+{% if already_enrolled %}
+<div class="panel panel--simple panel--info panel--spacious">
+    <div class="panel__body">
+        <div id="ALREADY_ADDED_NOTIF">You have already added that <a href="#new-survey">survey</a> </div>
```

Comment:
why is this ID all in caps? We tend to use snake_case for IDs.

Reply:
Just used the same format as "NEW_SURVEY_NOTIF" from the same file

Reply:
actually... we seem to use a random mixture

Reply:
We do.  There's no convention consistency within SDC - generally in frontend I'd expect `snake_case` or `camelCase`